lib/network_utils.ml

```diff
@@ -35,6 +35,7 @@ let modprobe = "/sbin/modprobe"
 let ethtool = ref "/sbin/ethtool"
 let bonding_dir = "/proc/net/bonding/"
 let fcoedriver = ref "/opt/xensource/libexec/fcoe_driver"
+let inject_igmp_query = "/usr/libexec/xenopsd/igmp_query_injector.py"
```

Comment:
This needs to be made configurable from the config file. See how this is done for the `fcoedriver` above.